src/caretogether-pwa/src/Referrals/Arrangements/ArrangementCard.tsx

```diff
@@ -326,24 +329,30 @@ export function ArrangementCard({ partneringFamily, referralId, arrangement, sum
             <AccordionSummary expandIcon={<ExpandMoreIcon />}
               sx={{ marginTop:1, paddingTop:1, backgroundColor: "#0000000a" }}>
               <Grid container>
-                <Grid item xs={4}>
+                <Grid item xs={3}>
                   <Badge color="success"
                     badgeContent={completedRequirementsWithContext.length}>
                     ✅
                   </Badge>
                 </Grid>
-                <Grid item xs={4}>
+                <Grid item xs={3}>
                   <Badge color="warning"
                     badgeContent={exemptedRequirementsWithContext.length}>
                     🚫
                   </Badge>
                 </Grid>
-                <Grid item xs={4}>
+                <Grid item xs={3}>
                   <Badge color="error"
-                    badgeContent={missingAssignmentFunctions + assignmentsMissingVariants + missingRequirementsWithContext.length}>
+                    badgeContent={missingAssignmentFunctions + assignmentsMissingVariants + missingRequirementsWithContext.length - upcomingRequirementsCount}>
                     ❌
                   </Badge>
                 </Grid>
+                <Grid item xs={3}>
+                  <Badge color="error"
```

Comment:
Suggest changing this to an 'information'-type color. See the documentation at https://mui.com/material-ui/react-badge/#color